content/content-strategy/cms/cms-requirements-checklist/index.yml

```diff
@@ -0,0 +1,20 @@
+pagetitle: CMS requirements checklist
+description: Get the CMS you need. Manage content the way you want. Use this checklist to clearly understand your own CMS requirements and optimisation needs.
+weight: 60
```

Comment:
I've seen this `weight` value elsewhere in the same PR. Can't recall where exactly. When two topics have the same value it will change the display order of the modules in the side nav. Good to keep an eye out and do a search-all for the value across topic pages. cc @Phend-117

Reply:
Steve. Done